internal/db/models/instance.go

```diff
@@ -36,6 +36,27 @@ const (
 	InstanceStatusTerminated
 )
 
+// PayloadStatus represents the state of a payload operation on an instance
+type PayloadStatus int
```

Comment:
Currently not all these statuses are being used because of the lack of fine grained tasks visibility. 

Only `PayloadStatusNode`, `PayloadStatusPendingCopy`, and `PayloadStatusExecuted` are used to signifying either no payload, payload present, payload final state after instance provisioning.